feat: Create 2022-07-25-una-stagione-di-incontri-con-la-community (#1285)
italia/developers.italia.it: _posts/it/2022-07-25-una-stagione-di-incontri-con-la-community.md

---
layout: post
title: "Una stagione di incontri con la community"
subtitle: "Community call e community lab dedicate ai servizi pubblici digitali: facciamo un bilancio insieme."
date: 2022-07-25 08:00:00 +0200
tags: developers designers communitycall communitylab
categories: news
author: Francesco Pognante
image: /assets/icons/logo-it.png
locale: it
---

> Altri mille di questi partecipanti! Si, perchè sono stati oltre mille le persone che hanno partecipato alla ricca stagione di eventi e occasioni di confronto organizzate da [Developers Italia](https://developers.italia.it/) e [Designers Italia](https://designers.italia.it/). In nove puntate abbiamo discusso assieme di design system del Paese, di sviluppo aperto per i servizi pubblici, eGovernment Benchmark, ma anche di strumenti e risorse pratiche, come Git, Bootstrap Italia 2.0, UI Kit, progettazione di API REST e l’uso container.

Reviewer: ```suggestion
Altri mille di questi partecipanti! Si, perché sono state oltre mille le persone che hanno partecipato alla ricca stagione di eventi e occasioni di confronto organizzate da [Developers Italia](https://developers.italia.it/) e [Designers Italia](https://designers.italia.it/). In nove puntate abbiamo discusso assieme di design system del Paese, di sviluppo aperto per i servizi pubblici, eGovernment Benchmark, ma anche di strumenti e risorse pratiche, come Git, Bootstrap Italia 2.0, UI Kit, progettazione di API REST e l’uso container.
```

> Durante queste occasioni abbiamo condiviso conoscenze, soluzioni e strumenti per migliorare i servizi pubblici digitali con i membri della community, favorendo l’incontro tra tecnici, designer, fornitori e funzionari pubblici. In sostanza, tra tutti coloro che in qualche modo partecipano alle creazione e al mantenimento di servizi pubblici digitali.

Reviewer: ```suggestion
Durante queste occasioni abbiamo condiviso conoscenze, soluzioni e strumenti per migliorare i servizi pubblici digitali con i membri della community, favorendo l’incontro tra tecnici, designer, fornitori e funzionari pubblici. In sostanza, tra tutti coloro che in qualche modo partecipano alla creazione e al mantenimento di servizi pubblici digitali.
```

> Tutto questo non sarebbe stato possibile senza la community di oltre 20.000 persone presenti sui canali [Forum Italia](https://forum.italia.it/) e [Slack Developers](https://slack.developers.italia.it/) composta da quanti si occupano di sviluppo, design, project management nelle PA, in house e aziende fornitrici . Un ringraziamento va quindi dedicato a loro, a chi ha partecipato agli eventi, ma anche a chi è intervenuto con un commento o un messaggio su Slack, a chi ha inviato una pull request su Github e a chi ha in qualche modo contribuito a far crescere a mantenere attiva questa comunità.

Reviewer: ```suggestion
Tutto questo non sarebbe stato possibile senza la community di oltre 20.000 persone presenti sui canali [Forum Italia](https://forum.italia.it/) e [Slack Developers](https://slack.developers.italia.it/) composta da quanti si occupano di sviluppo, design, project management nelle PA, in house e aziende fornitrici. Un ringraziamento va quindi dedicato a loro, a chi ha partecipato agli eventi, ma anche a chi è intervenuto con un commento o un messaggio su Slack, a chi ha inviato una pull request su GitHub e a chi ha in qualche modo contribuito a far crescere a mantenere attiva questa comunità.
```

Per rivivere la stagione che ci lasciamo alle spalle e recuperare qualche puntata persa, si possono rivedere i video dei community lab e delle community call sulla [playlist “community”](https://www.youtube.com/playlist?list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL) del canale YouTube del Dipartimento per la trasformazione digitale. 

### Tutti gli eventi

> COMMUNITY CALL

Reviewer: ```suggestion
## Community call
```

> ## Community call

Reviewer: ```suggestion
#### Community call
```

- L’importanza di un design system per i servizi pubblici digitali ([9 febbraio](https://developers.italia.it/it/news/2022/02/22/come-si-realizza-design-servizi-pubblici))
- eGovernment Benchmark: dall’Europa, i criteri minimi di qualità dei servizi pubblici digitali ([9 marzo](https://developers.italia.it/it/news/2022/03/18/come-eu-valuta-servizi-pubblici-digitali-egovernment-benchmark))
- Design: la sinergia possibile tra Università e Pubbliche amministrazioni ([6 aprile](https://developers.italia.it/it/news/2022/04/13/pa-universita-progettare-servizi-pubblici-insieme))
- Esperienze di sviluppo aperto per i servizi pubblici ([18 maggio](https://developers.italia.it/it/news/2022/05/20/sviluppo-open-source-migliorare-servizi-pubblici))

> COMMUNITY LAB

Reviewer: ```suggestion
## Community lab
```

> ## Community lab

Reviewer: ```suggestion
#### Community lab
```

> - L’uso di Git ([23 febbraio](https://www.youtube.com/watch?v=2Ph4VSyi9lU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL&index=9&t=2s))

Reviewer: ```suggestion
- L’uso di Git ([23 febbraio](https://www.youtube.com/watch?v=2Ph4VSyi9lU&list=PLLZQsns4g-N5Eu8xv0L-iYhGU-IWBQSTL&index=9))
```
